lib/ansible/modules/cloud/azure/azure_rm_storageblob.py

```diff
@@ -17,11 +17,11 @@
 DOCUMENTATION = '''
 ---
 module: azure_rm_storageblob
-short_description: Manage blob containers and blob objects.
+short_description: Manage Blob Containers and Blob objects
```

Comment:
Is "Blob" supposed to be capitalized? Should it be capitalized in the storage account docs too?

Reply:
@acozine Blob should be used as a term on short description and should be capitalized. What do you think?

Reply:
@Fred-sun I don't mind if it's capitalized or not, but whichever way we do it, the docs should be consistent - so if we capitalize Blob, we should always capitalize it (just like we always capitalize Azure).

Reply:
I'd vote for not capitalizing. The Azure website only capitalizes Azure Blob Storage. When they talk about "a blob" they don't capitalize. See  
https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-dotnet

Reply:
You are right, I will changed these! Thank you very much!